[See this exception on Errbit|<%= app_problem_url problem.app, problem %>]
<% if notice = problem.notices.first %>
h1. <%= notice.message %>
h2. Summary
<% if notice.request['url'].present? %>
h3. URL
[<%= notice.request['url'] %>|<%= notice.request['url'] %>]
<% end %>
h3. Where
<%= notice.where %>

h3. Occured
<%= notice.created_at.to_s(:micro) %>

h3. Similar
<%= (notice.problem.notices_count - 1).to_s %>

h2. Params
{code:title=Params|borderStyle=solid}
<%= pretty_hash(notice.params) %>
{code}

h2. Session
{code:title=Session|borderStyle=solid}
<%= pretty_hash(notice.session) %>
{code}

h2. Backtrace
{code:title=Backtrace|borderStyle=solid}
<% notice.backtrace.lines.each do |line| %><%= line.number %>:  <%= line.file_relative %> -> **<%= line.method %>**
<% end %>
{code}

<% if notice.env_vars.present? %>
h2. Environment
|| Key        || Value      ||
<% notice.env_vars.each do |key, val| %>| <%= key %> | <%= val %> |
<% end %>
<% end %>
<% end %>
